### ISSUE TITLE
After completing a word in the terminal, screen reader user cannot edit of review it

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#150376]
Reported via https://github.com/microsoft/vscode/issues/149882#issuecomment-1137054006

User is using orca.

> > > Another thing I can't do is complete a command in the terminal either.
> > Normally this is done via tab which screen readers often override. Do
> > you know how other terminals handle this? One workaround you could try
> > is to setup a custom keybinding to send the tab sequence to the shell:
> >
> > ```json
> > {
> > 	"key":"ctrl+.",
> > 	"command":"workbench.action.terminal.sendSequence",
> > 	"when":"terminalFocus",
> > 	"args":{  "text":"\u0009"  }
> > }
> > ```
> >
> > Also we plan on investing in shell integration-based auto complete
> > sometime in the future which will give you access to a autocomplete
> > similar to the editor, as opposed to shell-based.
>
> Pressing the tab key actually completes the command. The big problem for
> me is that I can't edit or review this command.

I'm guessing they can edit it but don't get the necessary feedback. Some things to look into:

- Can we improve announcing of shell-based tab completions, even if this is handled specially via an alert?
- Does left/right announce the character properly when moving the cursor?
- Are `Terminal: Run Recent Command` completions announced properly? This is probably what we will recommend in the future for screen reader users. Note that it doesn't do completion yet but we plan on adding that.

**Comment by meganrogge** [comment]
This should work now.

**Comment by amunger** [comment]
With NVDA I can successfully:
 - edit a commend from "run recent commands" with alt+enter
 - press tab to complete a filepath
 - use rightArrow to accept powershells ghost text

Is there something else to check? Was this specific to ORCA?

**Comment by meganrogge** [comment]
@amunger to verify, enable a screen reader.  have some program that provides autocomplete in the terminal. verify that when you press tab to accept a completion, then use arrow keys, the screen reader announces the characters.
